designer/client/src/components/graph/node-modal/editors/field/Input.tsx

```diff
@@ -38,17 +38,16 @@ export default function Input(props: InputProps): JSX.Element {
         <div className={className}>
             <div className={isMarked ? " marked" : ""}>
                 {
-                    <InputWithFocus
+                    <NodeInput
                         autoFocus={autoFocus}
                         readOnly={readOnly}
                         placeholder={placeholder}
                         onChange={onChange}
                         onFocus={onFocus}
                         type={type}
                         className={cx([
-                            !showValidation || allValid(validators, [formattedValue ? formattedValue : value])
-                                ? "node-input"
-                                : "node-input node-input-with-error",
+                            (!showValidation || !allValid(validators, [formattedValue ? formattedValue : value])) &&
```

Comment:
check this one one more time. in "compare" view is something wrong

Reply:
fix